en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md

```diff
@@ -17,45 +17,9 @@ built on open standards and common libraries, so lots of the techniques should
 feel familiar to you. This is just a quick run down to get you started
 with some special conventions.
 
-For a more practical-oriented approach to CMS customizations, refer to the
+For a more practical-oriented approach to CMS customizations, refer to
 [Howto: Extend the CMS Interface](/developer_guides/customising_the_admin_interface/how_tos/extend_cms_interface).
 
-## Installation
```

Comment:
Instead of updating this information I've removed it, since it doesn't belong in this section anyway.